iree/turbine/kernel/wave/thread_shape_analysis.py

```diff
@@ -68,6 +68,17 @@ def propagatable_op(node: fx.Node):
     )
 
 
+def propagate_resolutions(
+    custom_node: CustomOp, dst_op: CustomOp = None
+) -> list[fx.Node]:
+    propagated_resolutions = capture_forward_slice(custom_node.fx_node, propagatable_op)
+    if dst_op:
+        for node in propagated_resolutions:
+            get_custom(node).index = dst_op.index
```

Comment:
When would we use it without dst_op?

Reply:
Good question. This was when I was using it in 2 places where dst_op was optional. I can modify this so that it doesn't handle that case.